### PR TITLE
:sparkles: [Compose Multiplatform] License screen can now be opened.

### PR DESCRIPTION
**Pull request by Corvus400**
## Issue
- None.

## Overview (Required)
- As the title says.

## Links
- https://github.com/DroidKaigi/conference-app-2024/issues/813#issuecomment-2309812273

## Movie (Optional)
<video src="https://github.com/user-attachments/assets/30b01847-26c9-4df0-8768-c89f72e634b7" width="300" >

### CODE REVIEW

File: app-ios-shared/src/commonMain/kotlin/io/github/droidkaigi/confsched/shared/IosComposeKaigiApp.kt [hunk 1]
```diff
@@ -64,11 +64,18 @@ import platform.Foundation.NSURL
 import platform.UIKit.UIApplication
 import platform.UIKit.UIViewController
 
+private object ExternalNavControllerLink {
+    var onLicenseScreenRequest: (() -> Unit)? = null
+}
```
Comment (Corvus400) [inline app-ios-shared/src/commonMain/kotlin/io/github/droidkaigi/confsched/shared/IosComposeKaigiApp.kt:69]: I do not like this implementation very much. 🤔 
If there is a better way to implement it, we will follow it. 🫡

**Comment by Corvus400** [comment]
@takahirom @ry-itto 
Hmmm... the iOS build succeeds locally, but the build does not seem to succeed in CI. 🤔 
I would appreciate it if you could take a look. 🙇

**Comment by takahirom** [comment]
I think this is related. Could you check how the contributor screen is used?
https://github.com/DroidKaigi/conference-app-2024/blob/86ca73ab19643a605b971d20c4de9f50fb8e36dc/app-ios/Package.swift#L85

**Comment by takahirom** [comment]
I'm not sure if iOS has a dependency rule for this, like we shouldn't use the feature module from KMPClient. 👀

**Comment by Corvus400** [comment]
@takahirom 
I was able to pass the test! 🥳 
It really helped me because I had been struggling for hours! 😆

**Review by shin-usu — APPROVED** [review]
Looks Great To Me!

**Review by takahirom — APPROVED** [review]
Thank you for addressing this difficult issue!

**Pull request merged**